sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+
+    <!-- Taxonomic Scope -->
+    <property group='Taxonomic Scope' name='targetTaxonomicScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/targetTaxonomicScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:targetTaxonomicScope' dc:description='The taxonomic group(s) targeted for sampling during the dwc:Event.' comments='The dwc:Event to which the eco:targetTaxonomicScope refers could be at any level of hierarchy. In the case of a higher level (parent) dwc:Event, include all taxonomic groups surveyed in the child dwc:Events that contributed to the parent dwc:Event. Recommended best practice is to separate multiple values in a list with space vertical bar space ( | ).' examples='`Aves`; `Aves | Mammalia`; `Procellariformes`' required='false'/>
+    <property group='Taxonomic Scope' name='excludedTaxonomicScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/excludedTaxonomicScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:excludedTaxonomicScope' dc:description='The taxonomic group(s) explicitly excluded from sampling during the dwc:Event.' comments='The dwc:Event to which the eco:excludedTaxonomicScope refers could be at any level of hierarchy. In the case of a higher level (parent) dwc:Event, include all the taxonomic groups explicitly excluded from the child dwc:Events that contributed to the parent dwc:Event. Recommended best practice is to separate multiple values in a list with space vertical bar space ( | ).' examples='`Aves`; `Quercus`' required='false'/>
+    <property group='Taxonomic Scope' name='taxonCompletenessReported' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/taxonCompletenessReported' dc:relation='https://tdwg.github.io/eco/terms/#eco:taxonCompletenessReported' dc:description='Statement about whether the taxonomic completeness of the dwc:Event was assessed.' comments='This term is meant to alert users that the inventory was conducted in such a way that all of the target taxa (the combination of eco:targetTaxonomicScope and eco:excludedTaxonomicScope) should have been detectable if they were present during the dwc:Event. This term can provide data users with a qualitative measure of how comprehensively an area has been surveyed, which assists in interpreting species populations, areas of occupancy, inferring species absences, etc. This term is only relevant if the dwc:Event used restricted search or open search methods. If taxonomic completeness was assessed, the methods used or an explanation of the basis of the completeness should be stated in eco: taxonCompletenessProtocols. Recommended best practice is to use a controlled vocabulary.' examples='`not reported`; `reported complete`; `reported incomplete`' required='false'/>
```

Comment:
Without a required controlled vocabulary this becomes rather useless for any automated processing, doesn't it?
Should this extension have a controlled vocabulary for this field that is then enforced in the IPT?